multiqc/modules/cutadapt/cutadapt.py

```diff
@@ -100,6 +100,7 @@ def parse_cutadapt_logs(self, f):
         s_name = None
         end = "default"
         cutadapt_version = None
+        python_version = None
```

Comment:
I think we should only report versions of the scientific software, so python version probably is not important